R/makeDisplay.R

```diff
@@ -125,8 +125,11 @@ makeDisplay <- function(
   }
 
   if(verbose) message("* Validating 'panelFn'...")
-  panelEx <- kvApply(kvExample(data), panelFn)$value
-
+  if(!is.null(params) && class(params) == "list"){
```

Comment:
to be safe, use `inherits(params, "list")` instead of `class(params) == "list"` (class could be a vector)